deployment/volumes/uploads.yaml

```diff
@@ -9,4 +9,4 @@
       - ReadWriteOnce
     resources:
       requests:
-        storage: {{ .Values.uploadsStorage }}
+        storage: 25Gi
```

Comment:
But why we have changed this here if we don’t merge? @mattwr18

Reply:
I wasn't suggesting we shouldn't merge... I was suggesting that instead of just merging these changes, we have a quick copy of the actual values in `deployments`, `services`, `pvc`, etc. and make sure there are not other things missing.
Also, we could consider how it's organized. What _we_ actually use, and what is just for others who might want to use local file systems for uploads, for example.